src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,76 @@
 
 #include "t06_homework.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
-int t06_homework() {
+bool samewrd(string wrd1, string wrd2, int lgt){
+    bool ex = false;
+    bool wrn = false;
+    int a=0, b=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'b' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4593725)